src/loaders/assetLoader.js

```diff
@@ -129,7 +130,7 @@ module.exports = async function assetLoader() {
     this.emitFile(dest, item.content);
   });
 
-  let publicPath = `__webpack_public_path__ + ${JSON.stringify(path.join('/', assets, url))}`;
+  let publicPath = `__webpack_public_path__ + ${JSON.stringify(path.join('/', assets, url).replace(pathSepPattern, '/'))}`;
```

Comment:
> /local-cli/bundle/assetPathUtils.js#L38

That code is not relevant since it's part of RN packager and not haul.

Can you add a comment explaining it? It's not immediately obvious why we need to do it.